README.md

```diff
@@ -262,77 +201,220 @@ builder.toValue();
 
 ## API
 
-### Root
+### SQONBuilder default export
+`SQONBuilder(sqon: SQONBuilder | SQON | string)`
 
-The `SQONBuilder` is a function that will create a new builder from another SQONBuilder, SQON object, or JSON string.
+The package default export is the `SQONBuilder`. This is a function that will generate a `SQONBuilder` object from another `SQONBuilder`, a `SQON` object, or JSON `string`.
 
-Example: `const builder = SQONBuilder(sqonJson);`
+> Note: This will attempt to parse the provided string as JSON and will then validate that the contents are a valid SQON. If the provided string cannot be parsed from JSON a `SyntaxError` will be thrown. If the parsed string is not a valid SQON, a [`ZodError`](https://zod.dev/ERROR_HANDLING) will be thrown.
 
-All filters and combination methods described here can be called from the returned builder object.
+Example: `const builder: SQONBuilder = SQONBuilder({op: 'in', content: {fieldName: 'name', value: ['Jim']}});`
 
-Example: `builder.in('name','Jim').gt('score',95);`
+The `SQONBuilder` function also provides many static functions that can be used to generate a new SQONBuilder without having a SQON to start with.
 
-This will throw an error if the provided value is not a valid SQON or JSON string.
+Example: `const builder: SQONBuilder = SQONBuilder.in('name', 'Jim');`
 
-### Filters
+The `SQONBuilder` object that is returned stores the value of the generated SQON which can be accessed as a string with `.toString()` or as an object with `.toValue()`. From the `SQONBuilder` object you can now combine the stored SQON value with other filters or combination operations.
 
-#### SQON.in(fieldName, values)
+Example: `builder.lt('age','30').gt('score',95);`
 
-Creates a filter requiring the given field to have one of the given values. The value can be a single string or number value, or an array of strings or numbers.
 
-Example: `SQON.in('name',['Jim','Bob'])`
+#### Filter: In
+`SQONBuilder.in(fieldName: string, value: ArrayFilterValue) => SQONBuilder`
```

Comment:
this syntax seems a bit confusing (looks like an arrow function).
maybe I need some help reading it correctly?

Reply:
ah, reading the sentence bellow maybe this clears the example a little?
```suggestion
`SQONBuilder.in(fieldName: string, value: ArrayFilterValue)` returns a `SQONBuilder`
```